.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = true
```

Comment:
Will this affect code samples? e.g. for whitespace lang

Reply:
Does not in ``` blocks, does in inline blocks, annoyingly, but it's an avoidable (prettier can be disabled in a few different ways) edge-case.

Reply:
@wolf99 This block only targets markdown files. Erik [noted](https://github.com/exercism/v3/issues/863#issuecomment-613265983) we can use prettier's ignore comments since trailing whitespaces in markdown for formatting is rare.